source/sitemap.html.erb

```diff
@@ -0,0 +1,70 @@
+---
+title: Sitemap – GOV.UK Verify
+---
+
+<div class="govuk-width-container">
+  <nav aria-label="Breadcrumb" class="govuk-breadcrumbs">
+    <ol class="govuk-breadcrumbs__list">
+      <li class="govuk-breadcrumbs__list-item">
+        <a class="govuk-breadcrumbs__link" href="https://www.gov.uk/service-toolkit#components">Components</a>
+      </li>
+      <li class="govuk-breadcrumbs__list-item">
+        <a class="govuk-breadcrumbs__link" href="/">GOV.UK Verify</a>
+      </li>
+      <li class="govuk-breadcrumbs__list-item" aria-current="page">
+        <a href="#main">Sitemap</a>
+      </li>
+    </ol>
+  </nav>
+</div>
+
+<main class="govuk-main-wrapper" id="main" role="main">
+  <div class="govuk-width-container">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl">
+          Sitemap
```

Comment:
Sorry for the tiny content gripe – should be 'Site map' where it appears in the h1 and the title.

From what I've read, a site map is a HTML page that shows all the pages on the site, a sitemap is a XML file that lists URLS... 

Otherwise all looks good

Reply:
ah ok, will fix...